src/components/history/ReadRow.js

```diff
@@ -23,12 +23,22 @@ export default class ReadRow extends PureComponent {
     children: noop
   }
 
+  componentDidMount() {
+    this.onFocusListener = window.addEventListener('focus', () => {
```

Comment:
We have this abstraction for global events:

https://github.com/ubergrape/grape-web/blob/master/src/components/global-event/GlobalEvent.js